homeassistant/components/yale_smart_alarm/lock.py

```diff
@@ -45,7 +45,7 @@ def __init__(
     ) -> None:
         """Initialize the Yale Lock Device."""
         super().__init__(coordinator, data)
-        self._attr_code_format = f"^\\d{code_format}$"
+        self._attr_code_format = "^\\d{" + str(code_format) + "}$"
```

Comment:
Please use f-strings...

```suggestion
        self._attr_code_format = f"^\\d{{{code_format}}}$"
```